bin/http-server

```diff
@@ -80,6 +82,54 @@ else if (colors) {
     request: function () {}
   };
 }
+if (superverbose) {
+  logger = {
+    info: console.log,
+    request: function (req, res, error) {
+      var date = utc ? new Date().toUTCString() : new Date();
+      if (error) {
+        logger.info(
+          '[%s] "%s %s" Error (%s): "%s"',
+          date, colors.red(req.method), colors.red(req.url),
+          colors.red(error.status.toString()), colors.red(error.message)
+        );
+        logger.info('---------------header--------------');
+        var  request = req.request;
+        var rawHeaders = request.rawHeaders;
+        if (rawHeaders && rawHeaders.length > 0) {
+          var headerStr = '';
+          for (var i = 0; i < rawHeaders.length;i++) {
```

Comment:
Rather than a `for` loop, it might be cleaner and easier to use `rawHeaders.reduce` here. What do you think?